Race/race.py

```diff
@@ -0,0 +1,75 @@
+import random
+import time
+
+
+class Animal:
+
```

Comment:
let's define a method `has_won` in this class which will return a boolean telling if the animal position has reached the finish position.